coprocessor/fhevm-engine/transaction-sender/src/ops/verify_proof.rs

```diff
@@ -130,7 +130,7 @@ impl<P: alloy::providers::Provider<Ethereum> + Clone + 'static> VerifyProofOpera
                     warn!( "Coprocessor has already verified the proof with ID {}, removing it from the DB", txn_request.0);
                     self.remove_proof_by_id(txn_request.0).await?;
                     return Ok(());
-                } else if let Some(InputVerificationErrors::CoprocessorSignerAlreadyRejected(_)) =
+                } else if let Some(InputVerificationErrors::CoprocessorAlreadyRejected(_)) =
```

Comment:
@immortal-tofu check [line 125](https://github.com/zama-ai/fhevm/pull/463/files#diff-29a01c67e802a6948f7fecc5c698c3644403398b2974129bf96daabb8fd3b5b7R125) should be updated with new error name I guess